users/tasks.py

```
@@ -0,0 +1,108 @@
+            message=(
+                f"Hi {user.first_name or user.username},\n\n"
+                "Welcome to SpeakWise! Your account is ready.\n\n"
+                f"Get started: {dashboard_url}"
+            ),
+            from_email=settings.DEFAULT_FROM_EMAIL,
+            recipient_list=[user.email],
+            html_message=html_message,
+            fail_silently=False,
+        )
+        logger.info("Welcome email sent to user ID: %s", user.id)
+    except Exception as e:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)